### PR TITLE
Add superuser command 'list-rooms'

### PR DESCRIPTION
**Pull request by jmunhoz**
Patches to support an extended superuser command 'list-rooms'.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,19 @@ EOF
 ./matrix-bot -c test.cfg
 ```
 
+Notes:
```
Comment (psaavedra) [inline README.md:38]: Thanks for improve the README

File: matrixbot/matrix.py [hunk 2]
```diff
@@ -32,12 +32,16 @@ def __init__(self, settings):
...
         self.domain = self.settings["matrix"]["domain"]
         self.only_local_domain = self.settings["matrix"]["only_local_domain"]
+        self.super_users = settings["matrix"]["super_users"]
 
         self.subscriptions_room_ids = settings["subscriptions"].keys()
         self.revokations_rooms_ids = settings["revokations"].keys()
         self.allowed_join_rooms_ids = filter(lambda x: x != 'default', settings["allowed-join"].keys())
         self.default_allowed_join_rooms = settings["allowed-join"]["default"]
 
+        self.enable_list_rooms_commands = settings["commands"]["list-rooms"]["enable"]
```
Comment (psaavedra) [inline matrixbot/matrix.py:42]: Could you add an example for this in the  `matrix-bot.cfg.example` ?

**Review by psaavedra — CHANGES_REQUESTED** [review]
Other than that this MR looks great.

File: matrixbot/matrix.py (hunk 2 above)
Comment (jmunhoz) [reply matrixbot/matrix.py:42]: Sure. Added a minimal configuration working with the feat.

**Review by psaavedra — APPROVED** [review]

**Pull request merged**